src/eureka/lib/util.py

```diff
@@ -481,7 +482,7 @@ def get_mad(meta, log, wave_1d, optspec, optmask=None,
     return np.ma.mean(ediff)
 
 
-def get_mad_1d(data, ind_min=0, ind_max=-1):
```

Comment:
I also noticed that **ind_max** is set to -1 here by default. This would exclude the very last integration though because it's used like this: data[ind_min:ind_max]. So I changed this to **None**.

Reply:
Great catch!